gitcache/pkg/repourl.go

```diff
@@ -32,6 +32,10 @@ func (r *RepoURL) String() string {
 	return fmt.Sprintf("%s/%s/%s", r.Host, r.Owner, r.Repo)
 }
 
+func (r *RepoURL) NonURLString() string {
+	return r.Host + r.Owner + r.Repo
```

Comment:
do we want any seperator here, like underscore or hyphen? good in case we have overlaps.

Reply:
I can agree. I can add a separator.